### PR TITLE
85319502 fix saving ncr properties

### PR DESCRIPTION
**Pull request by phirefly**

### CODE REVIEW

File: app/models/cart.rb [hunk 1]
```diff
@@ -203,4 +204,10 @@ def import_cart_items(cart_items_params)
   def set_defaults
     self.status ||= 'pending'
   end
+
+  def deserialized_properties
+    properties_copy = {}
+    self.properties.collect{|p| properties_copy[p.property] = YAML::load(p.value)}
+    properties_copy
+  end
```
Comment (afeld) [inline app/models/cart.rb]: Maybe move this to the mixin?

File: app/views/communicart_mailer/_navigator_cart.html.erb [hunk 2]
```diff
@@ -63,14 +63,15 @@
...
           </tr>
-              <%- cart.properties.where.not(property: cart.property_exclusions).each do |cart_property| %>
-
-          <tr class="cart_item_information">
-            <td class="w-container results-list" align="left" colspan="2">
-              <strong><%= (cart_property.property).split(/(?=[A-Z])/).join(" ") %></strong>: <%= cart_property.value %>
-            </td>
-          </tr>
-              <%- end %>
+          <%- cart.deserialized_properties.each do |property,value| %>
```
Comment (afeld) [inline app/views/communicart_mailer/_navigator_cart.html.erb]: Can we pass in the decorated cart from the mailer?

Comment (phirefly) [inline app/views/communicart_mailer/_navigator_cart.html.erb]: Whaddya know, it was already decorated!

**Comment by phirefly** [comment]
Updated! Let me know if you guys see anything else.

**Pull request merged**